core/engine/src/builtins/temporal/instant/mod.rs

```diff
@@ -532,6 +533,12 @@ impl Instant {
             .with_message("not yet implemented.")
             .into())
     }
+
+    pub(crate) fn value_of(_this: &JsValue, _: &[JsValue], _: &mut Context) -> JsResult<JsValue> {
+        Err(JsNativeError::error()
```

Comment:
Should the error type not be `JsNativeError::typ`? The spec says so at least https://tc39.es/proposal-temporal/#sec-temporal.instant.prototype.valueof

Reply:
Yes, it should be. Huh, I could've sworn I typed `typ`. I'll switch those over.